config/robotvscode/data/user-data/User/settings.json

```diff
@@ -30,19 +30,11 @@
     "window.zoomLevel": 0,
     "editor.renderControlCharacters": false,
 
-    "http.proxy": "http://rb-proxy-apac.bosch.com:8080",
-    "https.proxy": "https://rb-proxy-apac.bosch.com:8080",
+    "http.proxy": "",
     "http.proxyAuthorization": null,
     "http.proxyStrictSSL": false,
     "editor.renderWhitespace": "all",
-    "robot.language-server.python": "${env:RobotPythonPath}/python3",
-    "robot.python.executable": "${env:RobotPythonPath}/python3",
```

Comment:
Hi Ngoan,
what is the effect of removing robot.python.executable and python.pythonPath?
Thank you,
 Thomas

Reply:
Hi Ngoan,
found the reason. settings are deprecated...
Thank you,
 Thomas

Reply:
Hi Thomas,

I have overlooked your comment.
Yes, they are deprecated. Please refer below detail:
- `robot.python.executable` is belong to `robotframework-lsp` extension which has been replaced by `robotcode`.
- `python.pythonPath` is deprecated python path configuration in the using VSCodium.

Thank you,
Ngoan